apps/src/StudioApp.js

```diff
@@ -98,15 +98,26 @@ var codegen = require('./lib/tools/jsinterpreter/codegen');
 var copyrightStrings;
 
 /**
- * The minimum width of a playable whole blockly game.
+ * Store experiment parameters.
  */
+const isBigPlayspaceExperiment = experiments.isEnabledAllowingQueryString(
```

Comment:
nit: `isBigPlayspaceEnabled`? Or `isBigPlayspaceExperimentEnabled`?

Reply:
Good call.  Let's use `isBigPlayspaceEnabled` since it can still apply if this graduates from being an experiment.